www/front_src/src/Resources/index.tsx

```diff
@@ -94,6 +96,12 @@ const Resources = (): JSX.Element => {
   >(null);
 
   const [loading, setLoading] = useState(true);
+  const [enabledAutorefresh, setEnabledAutorefresh] = useState(true);
+
+  const refreshIntervalMs = useSelector(
+    (state) => (state?.intervals?.AjaxTimeReloadMonitoring || 15) * 1000,
```

Comment:
Isn't it better to use a default value in the store directly?

Reply:
done